finesse/hardware/stepper_motor/stepper_motor_base.py

```diff
@@ -95,3 +103,21 @@ def move_to(self, target: Union[float, str]) -> None:
             raise ValueError("Angle must be between 0° and 270°")
 
         self.step = round(self.steps_per_rotation * target / 360.0)
+
+    def _move_to(self, target: Union[float, str]) -> None:
+        try:
+            self.move_to(target)
+        except Exception as error:
+            self.send_error_message(error)
```

Comment:
This is a counter intuitive construction. As `_move_to` is called by `pubsub` - which responds to an external caller -, that should be the public method, and the one called inside the `try` block would be the private method.

Reply:
The reason I did it this way is because I wanted `move_to` to be the interface to the class for "normal" use (i.e. if you just want to make an object and call a method on it). The private version of the method just adds error handling and you'd only ever really want it to be called by pubsub, because otherwise you won't see raised exceptions in the usual way.

Also it seemed more intuitive to me to have the child classes override public methods rather than private ones, which seems a little odd.

I'm not totally wedded to this though. What do you think?